platform/windows/display_server_windows.cpp

```diff
@@ -607,8 +607,11 @@ void DisplayServerWindows::show_window(WindowID p_id) {
 		_update_window_style(p_id);
 	}
 
-	ShowWindow(wd.hWnd, (wd.no_focus || wd.is_popup) ? SW_SHOWNOACTIVATE : SW_SHOW); // Show the window.
-	if (!wd.no_focus && !wd.is_popup) {
+	if (wd.no_focus || wd.is_popup) {
+		// NOTE: the equivalent to SW_SHOW is SW_SHOWNA, not SW_SHOWNOACTIVATE
```

Comment:
This is correct, but the comment is unnecessary.

Reply:
Sorry, I would have put in PR comment if I had just filed this one thing.  I was trying to explain why the change but that doesn't belong in the code and could be read as rude.

Reply:
Removed comment in amended commit.